qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -335,6 +337,14 @@ def analysis_results(
                 ``analysis_results(quality=[('ge', 'Bad'), ('lt', 'Good')])``
                 will return all analysis results with a quality of ``Bad`` and
                 ``No Information``.
+            tags: Filter by tags assigned to experiments.
+            tags_operator: Logical operator to use when filtering by job tags. Valid
+                values are "AND" and "OR":
+
+                    * If "AND" is specified, then an experiment must have all of the tags
+                      specified in `tags` to be included.
+                    * If "OR" is specified, then an experiment only needs to have any
```

Comment:
More experiment wording here but this isn't an experiment resource.

Reply:
Updated!